charts/federation-v2/values.yaml

```diff
@@ -19,18 +19,8 @@ controllermanager:
     requests:
       cpu: 100m
       memory: 64Mi
-  ## Feature gates are a set of key=value pairs that describe alpha or
-  ## experimental features. An administrator can use the --feature-gates
-  ## command line flag on each component to turn a feature on or off.
-  ## Federation V2 now have the following default feature gates:
-  ## PushReconciler=true,SchedulerPreferences=true,CrossClusterServiceDiscovery=true,FederatedIngress=true
-  featureGates:
-  ## The namespace the federation control plane is deployed in. If unset,
-  ## will default to federation-system.
-  federationNamespace:
-  ## The cluster registry namespace. If unset, will default to kube-multicluster-public.
-  registryNamespace:
-  
+    ## The cluster registry namespace. If unset, will default to kube-multicluster-public.
```

Comment:
Given the logic in `federationconfig.yaml`, would it be more correct to say:

```
...will default to kube-multicluster-public for a cluster-scoped deployment, and the federation namespace for namespace-scoped deployment.
```

Reply:
make sense